js/map.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+window.map = (function () {
+  var pinsFragment = renderPins();
+
+  function renderPins() {
+    var offers = window.data.generateOffers();
```

Comment:
offers лучше получать как значение аргумента функции
```suggestion
   function renderPins(offers) {
```